src/transformers/models/auto/configuration_auto.py

```diff
@@ -196,6 +199,7 @@
     [
         # Add full (and cased) model names here
         ("deit", "DeiT"),
+        ("luke", "Luke"),
```

Comment:
```suggestion
        ("luke", "LUKE"),
```
It seems Luke is written LUKE everywhere else (this is used to fill docstrings automatically, so should have the proper name of the model).